contract_isp/__openerp__.py

```diff
@@ -28,7 +29,8 @@
 Manage service based contracts
```

Comment:
README:
Please create a OCA README file:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
Transfer the description in there.
Follow the template for openerp.py: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py
version: 8.0.1.0.0